python/pyspark/sql/types.py

```diff
@@ -684,6 +702,28 @@ class StructType(DataType):
     ...     StructField("f2", IntegerType(), False)])
     >>> struct1 == struct2
     False
+
+    The below example demonstrates how to create a struct using class:`StructType`
+    and class:`StructField` on DataFrame:
```

Comment:
nit: `create a struct ... on DataFrame` may not be very clear, how about `create a DataFrame based on a struct created using ...`?

Reply:
Thanks, addressed.